packages/flutter/lib/src/material/outlined_button.dart

```diff
@@ -80,7 +80,7 @@ class OutlinedButton extends ButtonStyleButton {
     super.autofocus = false,
     super.clipBehavior = Clip.none,
     super.statesController,
-    required Widget super.child,
```

Comment:
So, this was changing the type of `child` from `Widget?` in the super-class to `Widget` in this class. With this change, OutlinedButton are now allowed to have a null child. Is that intended? 

/cc @HansMuller

Reply:
Currently TextButton requires a `Widget` child, and ElevatedButton's child is a `Widget?`. This may be due to sloppiness rather than thoughtful design. For the sake of consistency, it seems best to make them all `Widget?`.